theseus_gui/src/pages/Settings.vue

```diff
@@ -519,6 +524,19 @@ async function refreshDir() {
         />
       </div>
     </Card>
+    <Card>
+      <div class="label">
+        <h3>
+          <span class="label__title size-card-header">About</span>
+        </h3>
+      </div>
+      <div>
+        <label>
+          <span class="label__title">App version</span>
+          <span class="label__description">Theseus v{{ launcherVersion }} </span>
```

Comment:
This could easily be replaced by
```ts
import { version } from '../../package.json';
<span class="label__description">Theseus v{{ version }}</span>
```